### PR TITLE
Con 59 mg appsflyer connector

### PR DESCRIPTION
**Pull request by MarcGEthyca**
Closes #CON-59

### Description Of Changes

Adding a new Appsflyer Erasure Connector

### Code Changes

* [X] Added required files for configuration, dataset, fixtures, and testing for this connector
* [X] This connector also includes an override request

### Steps to Confirm

* [ ] _You should be able to see the Appsflyer option in the integration drop down from within Fides_

### Pre-Merge Checklist

* [x] All CI Pipelines Succeeded
* Documentation:
  * [X] documentation complete, [PR opened in fidesdocs](https://github.com/ethyca/fidesdocs/pulls)
 (https://github.com/ethyca/fidesdocs/pull/305)
  * [X] documentation [issue created in fidesdocs]
 (https://github.com/ethyca/fidesdocs/issues/304)
* [x] Issue Requirements are Met
* [x] Relevant Follow-Up Issues Created
* [x] Update `CHANGELOG.md`
* [X] For API changes, the [Postman collection](https://github.com/ethyca/fides/blob/main/docs/fides/docs/development/postman/Fides.postman_collection.json) has been updated

### CODE REVIEW

**Comment by MarcGEthyca** [comment]
Tests are all passing, this looks okay to me, hopefully no gremlins got into the code over the holidays =)

File: data/saas/config/appsflyer_config.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Appsflyer
```
Comment (galvana) [inline data/saas/config/appsflyer_config.yml:3]: The preferred casing is "AppsFlyer", make sure to update it whenever it's used as a label or in a sentence.

File: src/fides/api/service/saas_request/override_implementations/appsflyer_request_overrides.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    """
+    Gather the full list of applications set up, and retrieve the application name and
+    platform information for use with erasure endpoint.
+    """
+
+    user_ids = input_data.get("user_id")  #  lists
+    app_ids = input_data.get("app_id")  #  lists
+    # import pdb
+
+    # pdb.set_trace()
```
Comment (galvana) [inline src/fides/api/service/saas_request/override_implementations/appsflyer_request_overrides.py:32]: Remove all the commented out `pdb.set_trace()` once this is ready for review

File: tests/ops/integration_tests/saas/test_appsflyer_task.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+class TestAppsflyerConnector:
+    def test_connection(self, appsflyer_runner: ConnectorRunner):
+        appsflyer_runner.test_connection()
+
+    async def test_access_request(
+        self, appsflyer_runner: ConnectorRunner, policy, appsflyer_identity_email: str
+    ):
+        access_results = await appsflyer_runner.access_request(
+            access_policy=policy, identities={"email": appsflyer_identity_email}
+        )
```
Comment (galvana) [inline tests/ops/integration_tests/saas/test_appsflyer_task.py:19]: add an assertion here to verify that the user we return matches the identity email

**Review by galvana — CHANGES_REQUESTED** [review]
Overall looks good! Just a few minor things. I can test this once you add the secrets to 1password, I can't find the entry for AppsFlyer.

File: src/fides/api/service/saas_request/override_implementations/appsflyer_request_overrides.py (hunk 2 above)
Comment (MarcGEthyca) [reply src/fides/api/service/saas_request/override_implementations/appsflyer_request_overrides.py]: Added my saas_config.yml for AppsFlyer to 1password ([here](https://share.1password.com/s#rGCDpSJcKONHjushoxFqP4W0qsEJXMI0W3qFTSeYcu8)), as well as the account information in a [separate entry](https://share.1password.com/s#oUrPNpye257COix7RfNgr4Zwc44E7i3l4HtttC5kKGU), and lastly the API token ([here](https://share.1password.com/s#4Y6XXNB6dDleRiyTW2T-tPIz2RPyygHQpHPag3l_YuE)).

**Review by galvana — APPROVED** [review]

**Pull request merged**